gr-iio/include/gnuradio/iio/device_sink.h

```diff
@@ -61,7 +71,9 @@ class IIO_API device_sink : virtual public gr::sync_block
                      const std::vector<std::string>& params,
                      unsigned int buffer_size = DEFAULT_BUFFER_SIZE,
                      unsigned int interpolation = 0,
-                     bool cyclic = false);
+                     bool cyclic = false,
+                     bool tagged_input = false,
```

Comment:
Can tagged_input be implied by a non-blank len_tag_key? This would save a param/setter.

Reply:
I thought about that, but it would prevent us from providing a default tag name. Though i suppose that's not too much of an issue.